python/mxnet/gluon/data/sampler.py

```diff
@@ -136,3 +138,74 @@ def __len__(self):
         raise ValueError(
             "last_batch must be one of 'keep', 'discard', or 'rollover', " \
             "but got %s"%self._last_batch)
+
+
+class AliasMethodSampler(object):
```

Comment:
This is not a data sampler by gluon's definition.
It should be kept in the example code

Reply:
@piiswrong As the `NoiseContrastiveEstimationLoss` use it, where should I put the code?